ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -482,65 +485,83 @@ export const WorkspaceEdit = fp.flow(
           );
         }
       } else if (this.isMode(WorkspaceEditMode.Edit)) {
-        const fetchedBillingInfo = await getBillingAccountInfo(
-          this.props.workspace.googleProject
-        );
-        if (
-          !displayBillingAccounts.find(
-            (billingAccount) =>
-              billingAccount.name === fetchedBillingInfo.billingAccountName
-          )
-        ) {
-          // If the user has owner access on the workspace but does not have access to the billing account
-          // that it is attached to, keep the server's current value for billingAccountName and add a shim
-          // entry into billingAccounts so the dropdown entry is not empty.
-          //
-          // The server will not perform an updateBillingInfo call if the received billingAccountName
-          // is the same as what is currently stored.
-          //
-          // This can happen if a workspace is shared to another researcher as an owner.
+        try {
+          const fetchedBillingInfo = await getBillingAccountInfo(
```

Comment:
I'd prefer to localize the try/catch as much as possible.  This is the line causing the error, so can we scope it down to here only?

also, we may be able to use `Promise.catch()` here instead

Reply:
I switched this to Promise.catch instead of try/catch. Does the current approach seem better?